4001-5000/LIT4274AkkonuB.xml

```diff
@@ -6,7 +6,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <titleStmt>
             <title xml:lang="gez" xml:id="t1">አኮኑ፡ ብእሲ፡</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">ʾAkko-nu bǝʾsi</title>
-            <title xml:lang="en" corresp="#t1">Was he not a man?</title>
+            <title xml:lang="en" corresp="#t1">Is it not that a man…?</title>
```

Comment:
Can you add the deleted translation back possibly as t2? It is good to preserve labels used historically, and "Was he not a man" is the label used in EMML catalogues, by PEMM, by CANAMEi (https://www.ipocan.it/index.php/it/ms/123-parma-palatina-3835) etc

Reply:
Can I have two English translations of t1?

Reply:
(I added it as separate t2)